spharpy/spherical.py

```diff
@@ -472,7 +476,7 @@ def spherical_harmonic_basis_gradient(n_max, coordinates, normalization="n3d",
 
 
     This implementation avoids singularities at the poles using identities
-    derived in [#]_.
+    derived in [#]_ and [#]_.
```

Comment:
This is incorrect, the singularity avoiding formulation is detailed only in the second reference.

Reply:
@mberz I tried to fix it. But I'm not into the topic, if this isnt correct, or you have an better idea, feel free to change here in the PR directly